ci/azure/on_master_success

```diff
@@ -5,7 +5,7 @@ set +x
 
 set -e
 
-sudo apt-get update -y
+sudo apt-get update -y --allow-releaseinfo-change
```

Comment:
why is this required here but not here?
https://github.com/ziglang/zig/blob/e54c49be9b4b1a97805a3ffaab3f21bac48b8851/ci/azure/linux_script#L11